hubspot3/products.py

```diff
@@ -72,17 +71,22 @@ def _get_path(self, subpath: str):
     def create(self, data=None, **options):
         """Create a new product."""
         data = data or {}
-
-        # See: https://developers.hubspot.com/docs/methods/products/create-product
-        data = [{"name": name, "value": value} for name, value in data.items()]
```

Comment:
Note: I removed this as I felt that it was not a convention shared across the whole code base. I personally find more handy to be able to simply give a simple key / value `dict` but I wanted to be coherent with the rest of the code :)

Reply:
Hmm okay, that makes sense. I'll need to mention this change in the release notes